lib/src/components/Table/table.module.scss

```
@@ -51,7 +51,7 @@
       thead {
         background-color: var(--table-header-color-background-warm);
 
-        tr {
+        th {
```

Stylelint (17.14.1)

[error] Expected selector "th" (":is(:is(.content table) thead) th") to come before selector "th:has(> [aria-checked]:only-child)" (":is(.content table) th:has( > [aria-checked]:only-child)"), at line 46 (no-descending-specificity)

(no-descending-specificity)